app/views/flush.py

```diff
@@ -70,6 +70,7 @@ def _submit_data(user):
 
 
 def _get_user(decrypted_token):
-    user_id = UserIDGenerator.generate_id(decrypted_token)
-    user_ik = UserIDGenerator.generate_ik(decrypted_token)
+    id_generator = UserIDGenerator(current_app.config['EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS'])
```

Comment:
As we are constructing this twice with the same arguments i think we should construct it once on application startup.